1 Ejercicio

Para calcular el importe de IVA de 145 Euros es necesario multiplicar 145 por 0.21. Escribe en Python la expresión que calcula el IVA de 145 Euros.

In [1]:
145*0.21

30.45

Escribe en Python la expresión que calcula el IVA de 258 Euros.

In [2]:
258*0.21

54.18

Escribe una función en Python llamada calcula_iva que calcule el importe de IVA de una cierta cantidad  x.

In [3]:
def calcula_iva(x):
    y=x*0.21
    return y

Ejecuta la función calcula_Iva para las cantidades 145 y 258. Comprueba que los resultados son 30.45 y 54.18

In [4]:
calcula_iva(145)


30.45

In [5]:
calcula_iva(258)

54.18

2 Ejercicio

Evalúa el polinomio en x4+x3+1/2x2−x
   x=10. El resultado ha de ser un número real.

In [7]:
def polinomio(x):
    z=(x**4)+(x**3)+(1/2)*(x**2)-x
    return z

In [9]:
polinomio(10)

11040.0

In [10]:
polinomio(7)

2761.5

3 Ejercicio

Define una función llamada raiz_cubica que devuelva el valor de x**(1/3). 

In [11]:
def raiz_cubica (x):
    raiz_cubica=x**(1/3)
    return raiz_cubica

In [12]:
raiz_cubica(5)

1.7099759466766968

Escribe la expresión Python que construya la siguiente cadena:
          "El resultado de la raíz cúbica de 5 es:1.70"
 
Utiliza el operador de formato %f.

In [13]:
cadena=("El resultado de la raíz cúbica de %.3f es: %.3f")
cadena % (5,1.70)

'El resultado de la raíz cúbica de 5.000 es: 1.700'